react/states_and_effects/introduction_to_state.md

```diff
@@ -0,0 +1,101 @@
+### Introduction
+
+Any exciting application you build is likely to change over its lifetime. The change could be as simple as toggling a dropdown menu or as complex as fetching data from an API. React provides primitives to manipulate state of your apps, more specifically components, to make them dynamic. In this lesson, we will learn how to use state in React.  
```

Comment:
`1.` lifetime. __The change__ could be `to` lifetime. __Changes__ could be
`2.` to manipulate __state__ of your `to` to manipulate __the state__ of